_includes/initiative.html

```diff
@@ -1,7 +1,7 @@
 {% if include.initiative == "Customer Experience" %}
     <a href="{{site.baseurl}}/coe/customer-experience.html" class="article-coe-link">CUSTOMER EXPERIENCE</a>
 {% elsif include.initiative == "Centers of Excellence" %}
-    <a href="{{site.baseurl}}/coe/centers-of-excellence.html" class="article-coe-link">CENTERS OF EXCELLENCE</a>
+    <a href="{{site.baseurl}}/about/approach-team-structure.html#our-approach" class="article-coe-link">CENTERS OF EXCELLENCE</a>
```

Comment:
[question] this seems like an unrelated change, if it's supposed to be included, can you call out why it changed in the PR description?